Update presence-admins.md (#6689)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/presence-admins.md

|| ![Open gray circle, indicates status unknown](media/Presence_Unknown.png) Status unknown|
|| ![Purple circle with arrow, indicates Out of office](media/Presence_OOF.png) Out of Office. Out of Office is used when an automatic reply is set. (Available in Outlook only.) |
|||

>  > [!Note]

Reviewer: ```suggestion
> [!NOTE]
```

>  > For users that there mailbox is hosted onprem it is expected to have presence delays with a maximun of an hour.

Reviewer: ```suggestion
 > Users whose mailboxes are hosted on-premises can expect to experience presence delays of a maximum of one hour.
```
